app/views/admin/loans/_form.html.slim

```diff
@@ -16,6 +16,10 @@
   = authorized_form_field(simple_form: f, model: @loan, field_name: :organization,
     choices: @organization_choices, include_blank_choice: false, classes: 'hidden-print')
 
+  = f.attribute :id
+    .static-text-as-field
+      = @loan.id
```

Comment:
Please add this to the en.yml file as `ID`. Then it will get translated along with the other attribs, and I think ID looks better than Id (unless you're talking about Freudian psych :))